traefik/rootfs/etc/traefik/traefik.yaml.gotmpl

```diff
@@ -20,6 +20,9 @@ entryPoints:
 api:
   dashboard: true
   insecure: true
+  
+pilot:
+    token: {{ $options.pilot_token }}
```

Comment:
Please adhere to this file's indentation which is 2 spaces.

Reply:
Yes, my bad, I was quickly editing via the browser and didn't noticed it. I fixed it.